File: VotacaoAlterData.Domain/Entity/ItemRecurso.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace VotacaoAlterData.Domain.Entity
{
    public class ItemRecurso
    {
        public int ItemRecursoId { get; set; }
        public string Descricao { get; set; }
        public DateTime DataCadastro { get; set; }
        public int RecursoId { get; set; }
        public virtual Recurso Recurso { get; set; }
        public virtual List<Voto> Votos { get; set; }

    }
}

File: VotacaoAlterData-App/src/app/_models/ItemRecurso.ts
import { Voto } from "./Voto";
import { Recurso } from './Recurso';
import { ItemRecursoVoto } from "./ItemRecursoVoto";

export class ItemRecurso
{
  itemRecursoId: number;
  dataCadastro: Date;
  recursoId: string;
  descricao: string;
  active: boolean = false;
  recurso: Recurso;
  votado: boolean;
  votos: Voto[];

}
